api/v1alpha1/loadbalancer_types.go

```diff
@@ -58,7 +58,7 @@ type ConsistentHash struct {
 
 	// The table size for consistent hashing, must be prime number limited to 5000011.
```

Comment:
can we add CEL to validate it must be prime number?

Reply:
Seems like CEL in Kubernetes [^1] doesn't support complex validation logic like `IsPrime` function. But we can validate whether table size is prime number when translating ClientTrafficPolicy to IR.

```go
func IsPrime(n int) bool {
    if n <= 1 {
        return false
    }
    for i := 2; i*i <= n; i++ {
        if n%i == 0 {
            return false
        }
    }
    return true
}
```

[^1]: https://kubernetes.io/docs/reference/using-api/cel/#cel-options-language-features-and-libraries

Reply:
SGTM